tests/core/api/GH_ORG_SUBS_API.js

```diff
@@ -0,0 +1,436 @@
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
+        assert.isNotEmpty(sub,
+          'Subscriptions cannot be empty.');
+        ownerApiAdapter.getRunStatusBySubscriptionId(sub.id,
+          function (err, runStas) {
+            if (err || _.isEmpty(runStas))
+              return done(
+                new Error(
+                  util.format('User cannot get run status',
+                    sub.id, err)
+                )
+              );
+
+            runStatus = runStas;
+            assert.isNotEmpty(runStatus, 'User cannot find the run status by subscription ID');
+            return done();
+          }
+        );
+      }
+    );
+
+    it('13. Member can get Run Status by subscription ID',
+      function (done) {
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
+        assert.isNotEmpty(sub,
+          'Subscriptions cannot be empty.');
+        memberApiAdapter.getRunStatusBySubscriptionId(sub.id,
+          function (err, runStas) {
+            if (err || _.isEmpty(runStas))
+              return done(
+                new Error(
+                  util.format('User cannot get run status',
+                    sub.id, err)
+                )
+              );
+
+            runStatus = runStas;
+            assert.isNotEmpty(runStatus, 'User cannot find the runstatus by subscription ID');
+            return done();
+          }
+        );
+      }
+    );
+
+    it('14. Collaborator can get Run Status by subscription ID',
+      function (done) {
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
+        assert.isNotEmpty(sub,
+          'Subscriptions cannot be empty.');
+        collaboraterApiAdapter.getRunStatusBySubscriptionId(sub.id,
+          function (err, runStas) {
+            if (err || _.isEmpty(runStas))
+              return done(
+                new Error(
+                  util.format('User cannot get run status',
+                    sub.id, err)
+                )
+              );
+
+            runStatus = runStas;
+            assert.isNotEmpty(runStatus, 'User cannot find the runstatus by subscription ID');
+            return done();
+          }
+        );
+      }
+    );
+
+    it('15. unauthorized user cannot get Run Status by subscription ID',
+      function (done) {
+        var sub = _.findWhere(subscriptions, {orgName:global.TEST_GH_ORGNAME}) || {};
+        assert.isNotEmpty(sub,
+          'Subscriptions cannot be empty.');
+        unauthorizedApiAdapter.getRunStatusBySubscriptionId(sub.id,
+          function (err, runStas) {
+            assert.strictEqual(err, 404,
+              util.format('User should not be able to get the runstatus by subscription ID. ' +
+                'err : %s %s', err, runStas)
+            );
+            return done();
+          }
+        );
+      }
+    );
+
+    it('16. public user cannot get Run Status by subscription ID',
```

Comment:
we also need to test the public project where the unauthorized and public guys can get this information correct?